lib/windows/winlean.nim

```diff
@@ -350,6 +350,13 @@ else:
 proc findClose*(hFindFile: Handle) {.stdcall, dynlib: "kernel32",
   importc: "FindClose".}
 
+type LPPROGRESS_ROUTINE* = proc(TotalFileSize, TotalBytesTransferred, 
+  StreamSize, StreamBytesTransferred: int64, dwStreamNumber, 
```

Comment:
types are wrong, you should use the types as they appear in windows docs (LARGE_INTEGER)
https://docs.microsoft.com/en-us/windows/win32/api/winbase/nc-winbase-lpprogress_routine
(depending on 32 vs 64 bit, this may be wrong)

Reply:
It's not defined in `winlean`. I've read this:
https://docs.microsoft.com/en-us/windows/win32/api/winnt/ns-winnt-large_integer-r1

Probably I'm wrong, but I don't know how to fix it.